integration-tests/tests/src/tests/sync/asymmetric.ts

```diff
@@ -78,6 +78,7 @@ describe.skipIf(environment.missingServer, "Asymmetric sync", function () {
     it("an asymmetric schema cannot be queried through '_objectForObjectKey()'", function (this: RealmContext) {
       expect(() => {
         // A valid objectKey is not needed for this test
+        //@ts-expect-error Should not be correct usage
```

Comment:
Not sure if `Internal method` may be more appropriate ts expect err message here? (Like the other ones you added). Since from TS's side, the error is regarding accessing an internal method, while the test itself already describes the incorrect usage part.